expenses-manager-backend/src/main/java/com/endava/expensesmanager/controller/UsersController.java

```diff
@@ -37,6 +37,12 @@ public ResponseEntity<Users> getUserById(@PathVariable int id) {
         return user.map(ResponseEntity::ok).orElseGet(() -> ResponseEntity.notFound().build());
     }
 
+    @GetMapping("/byName")
```

Comment:
The naming convention used for path is "Kebab Case". See point 2.2.3 for an example https://restfulapi.net/resource-naming/